### PR TITLE
Theme/multisite 13187

### PR DESCRIPTION
**Pull request by MehdiDonk**

### CODE REVIEW

File: lib/themes/erasmus/scripts/erasmus_guide.js [hunk 1]
```diff
@@ -3,169 +3,169 @@
...
+        $('body').append(toolbar);
+
+      }
+      else {
+        var bookNav = '';
+      }
+
+      $(window).scroll(function () {
+        if ($(".book-navigation").isOnScreen() == true || $('header.main-banner').isOnScreen() == true) {
+        $(".book-toolbar").removeClass('show-toolbar');
```
Comment (jonhy81) [inline lib/themes/erasmus/scripts/erasmus_guide.js]: Fix indentation here too.

File: lib/themes/erasmus/templates/block/block--sidebar-left.tpl.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+<?php print render($title_prefix); ?>
+<?php if ($title && $block->subject) : ?>
+  <h2>
+    <?php print $block->subject ?>
+  </h2>
+<?php endif;?>
+<?php print render($title_suffix); ?>
+
+  <div class="content"<?php print $content_attributes; ?>>
```
Comment (jonhy81) [inline lib/themes/erasmus/templates/block/block--sidebar-left.tpl.php]: Fix indentation here.

File: lib/themes/erasmus/templates/page--programme-guide.tpl.php [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    <?php if ($regions['sidebar_left']) : ?>
+     <aside class="col-sm-3 sidebar-left">
+        <div class="sidebar-wrapper">
+          <?php print $regions['sidebar_left']; ?>
+        </div>
+      </aside>
+    <?php endif; ?>
+
+    <section class="main-content-wrapper col-sm-8" role="main">
+  <?php
```
Comment (jonhy81) [inline lib/themes/erasmus/templates/page--programme-guide.tpl.php]: Move this snippet to a preprocess function and then in the page print the new var. Do not put code in your tpl.

File: lib/themes/erasmus/templates/block/block--sidebar-left.tpl.php (hunk 2 above)
Comment (MehdiDonk) [inline lib/themes/erasmus/templates/block/block--sidebar-left.tpl.php]: Ok I will fix this.
I haven't check this part as this file hasn't changed with the last pull request.

File: lib/themes/erasmus/templates/page--programme-guide.tpl.php [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        $views_page = views_get_page_view();
+        if (is_object($views_page)) {
+            print '<h1>' . $title . '</h1>';
+        }
+  ?>
+      <a id="content"></a>
+
+          <?php if ($messages) : ?>
+        <div id="messages">
```
Comment (jonhy81) [inline lib/themes/erasmus/templates/page--programme-guide.tpl.php]: Also indentation here is not right

File: lib/themes/erasmus/templates/page--programme-guide.tpl.php [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+            <?php print $messages; ?>
+        </div>
+          <?php endif; ?>
+        <?php if ($regions['help']) : ?>
+        <div class="tabs">
+        <?php print $regions['help']; ?>
+              </div>
```
Comment (jonhy81) [inline lib/themes/erasmus/templates/page--programme-guide.tpl.php]: Check indentation in all file, please. Always use 2 spaces.

File: lib/themes/erasmus/templates/page--programme-guide.tpl.php (hunk 3 above)
Comment (MehdiDonk) [inline lib/themes/erasmus/templates/page--programme-guide.tpl.php]: Indeed! Working on it.

File: lib/themes/erasmus/templates/page--programme-guide.tpl.php (hunk 4 above)
Comment (MehdiDonk) [inline lib/themes/erasmus/templates/page--programme-guide.tpl.php]: Fixed (I verified the whole page and it should be fine by now)

File: lib/themes/erasmus/templates/page--programme-guide.tpl.php (hunk 5 above)
Comment (MehdiDonk) [inline lib/themes/erasmus/templates/page--programme-guide.tpl.php]: Fixed

**Pull request merged**